commit bb608e9db7d29616fb6e0d856c23434610d4a1bd
Author: Senthil Balasubramanian <senthilkumar@atheros.com>
Date:   Thu Dec 4 20:38:13 2008 +0530

    wireless: Incorrect LEAP authentication algorithm identifier.
    
    This patch fixes a regression introduced by
    "wireless: avoid some net/ieee80211.h vs. linux/ieee80211.h conflicts"
    LEAP authentication algorithm identifier should be 128.
    
    Signed-off-by: Senthil Balasubramanian <senthilkumar@atheros.com>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

diff --git a/include/linux/ieee80211.h b/include/linux/ieee80211.h
index a6ec928186ad..c4e6ca1a6306 100644
--- a/include/linux/ieee80211.h
+++ b/include/linux/ieee80211.h
@@ -836,7 +836,7 @@ struct ieee80211_ht_info {
 /* Authentication algorithms */
 #define WLAN_AUTH_OPEN 0
 #define WLAN_AUTH_SHARED_KEY 1
-#define WLAN_AUTH_LEAP 2
+#define WLAN_AUTH_LEAP 128
 
 #define WLAN_AUTH_CHALLENGE_LEN 128